src/arch/riscv/vmm.c

```diff
@@ -21,6 +21,24 @@ void vmm_arch_init()
     CSRW(CSR_HIDELEG, HIDELEG_VSSI | HIDELEG_VSTI | HIDELEG_VSEI);
     CSRW(CSR_HEDELEG, HEDELEG_ECU | HEDELEG_IPF | HEDELEG_LPF | HEDELEG_SPF);
 
+    /**
+     * Enable and sanity check presence of Sstc extension if the hypervisor was
+     * configured to use it (via the CPU_EXT_SSTC macro). Otherwise, make sure
+     * it is disabled.
+     */
+    if (CPU_HAS_EXTENSION(CPU_EXT_SSTC)) {
+        CSRS(CSR_HENVCFG, HENVCFG_STCE);
+        bool sttc_present = (CSRR(CSR_HENVCFG) & HENVCFG_STCE) != 0;
```

Comment:
there is a typo here sttc_present should be sstc_present.

Reply:
fixed!

Reply:
fixed!